editor/multi_node_edit.cpp

```diff
@@ -49,6 +49,11 @@ bool MultiNodeEdit::_set_impl(const StringName &p_name, const Variant &p_value,
 		name = "script";
 	}
 
+	Node *node_path_target = nullptr;
+	if (p_value.get_type() == Variant::NODE_PATH) {
+		node_path_target = es->get_node(p_value);
+	}
+
```

Comment:
What happens when node_path_target is accessed when null?

Reply:
It spams errors when used later in `get_path_to(node_path_target)`.
Actually I've realized it myself before reading your comment that `p_value` can be a `NodePath()` (it happens when clearing the path) which would lead to an error itself (in `es->get_node(p_value)`) and to `node_path_target` being null resulting in even more errors. Just pushed a fix.